easybuild/easyconfigs/m/MAJIQ/MAJIQ-1.1.1-intel-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,75 @@
+easyblock = 'Bundle'
+
+name = 'MAJIQ'
+commit = '9671da7'
+version = '1.1.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://majiq.biociphers.org'
+description = """MAJIQ and Voila are two software packages that together detect, quantify, and visualize
+ local splicing variations (LSV) from RNA-Seq data."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    # older version of Cython required to work around "AttributeError: ... has no attribute '__reduce_cython__'
+    # see https://github.com/cython/cython/issues/1953
```

Comment:
Unless I am misinterpreting this comment (in which case it should be rephrased), since `python-3.6.4-intel-2018a` comes bundled with `Cython-0.27.3`, there should be no need to carry over the older version of `Cython` as builddependency.
Can you clarify this?

Reply:
`0.27.3` is too new, `0.25.2` doesn't have the issue

Reply:
Maybe add another comment line just to make it a little more explicit?

Reply:
Or, even better in my opinion, adding it to the `MAJIQ` bundle so that there is less risk of user conflict with the one bundled with `Python`?

Reply:
Moving it into the bundle makes it a runtime dependency, which it's not...

I'll clarify the comment.